notification/src/main/java/com/chronica/notification/data/entity/Invitation.java

```diff
@@ -0,0 +1,17 @@
+package com.chronica.notification.data.entity;
+
+import jakarta.persistence.Entity;
+import lombok.*;
+import java.time.LocalDateTime;
+
+@EqualsAndHashCode(callSuper = true)
+@Entity
+@RequiredArgsConstructor
+@Data
+public class Invitation extends Notification {
+    private Long invitationFromUserId;
```

Comment:
Ta nazwa na pewno jest zła. Jeśli chcesz utrzymać konwencję z notification to powinno być inviterId, ale ja jestem raczej za tym żeby to było userFromId/invitingUserId - jeśli użyjesz którejś z moich propozycji to musisz zmienić nazwę recieverId w notification